notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1310_Long_Tracking_Drift_06Apr24.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<p>Maybe reduce the font size of this comment so it doesn't look like a huge heading, which actually makes it a bit less readable in my opinion.</p><p> </p><p>Can you specify exactly how to retrieve this file? (directory or web path to RubinTV metadata to download)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>